### DISCUSSION TITLE
Invite for Discord channel

### DISCUSSION THREAD

**Discussion by Schmedu** [johnlindquist/kit#871] [General]
Hey @johnlindquist, thanks for creating ScriptKit - I love it! I would like to be part of the Discord channel and I saw you attached an invite to the latest release post but it expired unfortunately. Could you please send a new one?

I'm probably not the only one missing out on that 😅

**Comment by johnlindquist** [comment]
@Schmedu here you go :)

https://discord.gg/2KcPMkhx

**Reply by bigloudjeff** [reply]
Do you happen to have another one?

**Reply by johnlindquist** [reply]
@bigloudjeff Here you go: https://discord.gg/fVQxZPcV

**Reply by bigloudjeff** [reply]
Thanks!

**Comment by gavriguy** [comment]
Hi looks like the new link is also expired. Can you please share a new one?

**Reply by johnlindquist** [reply]
@gavriguy here you go: https://discord.gg/SJDg5nsr

**Comment by johtso** [comment]
@johnlindquist is there still a discord channel? any chance I could get an invite? ✨

**Reply by johnlindquist** [reply]
@johtso Here you go:

https://discord.gg/3xdhG5fyJM
